File: 快速发药机程序/OrderPrint/OrderPrint/BasketStock.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace OrderPrint
{
    class BasketStock
    {
        private string checkCode;
        private string lightCode;
        private string closeCode;
        private bool basket;
        private bool light;

        public string CheckCode
        {
            set { checkCode = value; }
            get { return checkCode; }
        }

        public string LightCode
        {
            set { lightCode = value; }
            get { return lightCode; }
        }

        public string CloseCode
        {
            set { closeCode = value; }
            get { return closeCode; }
        }

        public bool Basket
        {
            set { basket = value; }
            get { return basket; }
        }

        public bool Light
        {
            set { light = value; }
            get { return light; }
        }

    }
}

File: UnServer/src/us/BasketStock.java
package us;

public class BasketStock {
	private String cacheNO;
	private String checkCode;
	private String lightCode;
	private String closeCode;
	private boolean basket;
	private boolean light;

	public String getCacheNO() {
		return cacheNO;
	}

	public void setCacheNO(String cacheNO) {
		this.cacheNO = cacheNO;
	}

	public String getCheckCode() {
		return checkCode;
	}

	public void setCheckCode(String checkCode) {
		this.checkCode = checkCode;
	}

	public String getLightCode() {
		return lightCode;
	}

	public void setLightCode(String lightCode) {
		this.lightCode = lightCode;
	}

	public String getCloseCode() {
		return closeCode;
	}

	public void setCloseCode(String closeCode) {
		this.closeCode = closeCode;
	}

	public boolean isBasket() {
		return basket;
	}

	public void setBasket(boolean basket) {
		this.basket = basket;
	}

	public boolean isLight() {
		return light;
	}

	public void setLight(boolean light) {
		this.light = light;
	}
}
